lib/src/dropdown/dropdown.component.scss

```diff
@@ -0,0 +1,332 @@
+                box-shadow: none;
+                background-color: white;
+            }
+        }
+
+        &.more {
+            background-color: inherit !important;
+            height: $dropdown-toggle-height !important;
+            min-width: $dropdown-toggle-height !important;
+            max-width: $dropdown-toggle-height !important;
+            padding: 0px;
+            border-color: transparent;
+            margin: 0px 0px 0px auto;
+            > .right-items > .dropdown-icon-holder > svg {
+                height: calc(#{$dropdown-toggle-height} / 1.2);
+                width: calc(#{$dropdown-toggle-height} / 1.2);
+                &:focus {
+                    &:active {
+                        fill: white;
+                    }
+                }
+            }
+
+            &:hover {
+                background-color: $blue !important;
+                > .right-items > .dropdown-icon-holder > svg {
+                    fill: white;
+                }
+            }
+
+            &:focus {
+                border-color: $blue-darker;
+                &:active {
+                    background-color: $blue-darker !important;
+                }
+            }
+
+            &.open {
+                background-color: $blue-darker !important;
+                > .right-items > .dropdown-icon-holder > svg {
+                    fill: white;
+                }
+            }
+        }
+    }
+
+    > .custom-dropdown-menu {
+        min-width: $dropdown-width;
+        border-radius: 4px;
+        border: 1px solid $blue-darker;
+        padding: 0px;
+        margin: 4px 0px 14px 0px;
+        outline: none;
+        // max-height: 40vh;
+        overflow: auto;
+
+        > .dropdown-divider {
+            margin: 0px;
+
+            &.blue {
+                border-color: $blue-darker;
+            }
+        }
+
+        > .search-input {
+            height: $dropdown-toggle-height;
+            padding: 0px 0px 0px 16px;
+            border: none;
+            outline: none;
+        }
+
+        > .custom-dropdown-item {
+            display: flex;
+            padding-left: 16px;
+            background: white;
+            color: $body-color;
+            > label {
+                &:not(.form-check-label),
+                &:not(.custom-checkbox),
+                &:not(.custom-radio),
+                &:not(.custom-control-label) {
+                    cursor: pointer;
+                    color: $body-color;
+                    font-weight: normal;
+                    margin-bottom: 0px;
+                    line-height: initial;
+                }
+            }
+            &:focus {
+                outline: none;
+                background: initial;
+                color: initial;
+                border: initial;
+            }
+
+            &:hover {
+                background: initial;
+                color: initial;
+                border: initial;
+            }
```

Comment:
`initial` is [not supported in IE](https://developer.mozilla.org/en-US/docs/Web/CSS/initial#Browser_compatibility)

Reply:
setting it to inherit will have a different visual effect on the component so I will leave it for now. Anyway... IE support is not a priority